filesystems/gcs/src/test/scala/cromwell/filesystems/gcs/GcsPathBuilderSpec.scala

```diff
@@ -345,10 +345,10 @@ class GcsPathBuilderSpec extends TestKitSuite with FlatSpecLike with Matchers wi
 
   private def badPaths = Seq(
     BadPath("an empty path", "", " does not have a gcs scheme"),
-    BadPath("an bucketless path", "gs://", "Expected authority at index 5: gs://"),
-    BadPath("a bucket named .", "gs://./hello/world", "gs://./hello/world does not have a host"),
+    BadPath("an bucketless path", "gs://", "The specified GCS path 'gs://' does not parse as a URI.\nExpected authority at index 5: gs://"),
+    BadPath("a bucket named .", "gs://./hello/world", "The path 'gs://./hello/world' does not seem to be a valid GCS path. Please check that it starts with gs:// and that the bucket and object follow GCS naming guidelines at https://cloud.google.com/storage/docs/naming."),
     BadPath("a non ascii bucket name", "gs://nonasciibucket£€/hello/world",
-      "gs://nonasciibucket%C2%A3%E2%82%AC/hello/world does not have a host"),
+      "The path 'gs://nonasciibucket£€/hello/world' does not seem to be a valid GCS path. Please check that it starts with gs:// and that the bucket and object follow GCS naming guidelines at https://cloud.google.com/storage/docs/naming."),
```

Comment:
So actually it might not be that simple - the `pathAsString` method calls `toUri` to get back a gcs path with the `gs://`, which fails because the bucket contains an underscore...
So we can create a valid CloudStorageFilesystem / CloudStoragePath but we can't get it back to a usable String using `toUri`...